.github/workflows/rust.yml

```diff
@@ -29,4 +29,6 @@ jobs:
     - name: Rustfmt
       run: cargo fmt --check
     - name: Verify regenerated files
-      run: ./scripts/unicode.py && diff tables.rs src/tables.rs
\ No newline at end of file
+      run: ./scripts/unicode.py && diff tables.rs src/tables.rs
```

Comment:
sweet, thanks for adding this. I've been adding this for the other unicode- crates bit by bit